src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_ExcelCompat.txt

```diff
@@ -197,4 +197,236 @@ Table({Value:"08:03:05"},{Value:"00:00:00"})
 "1899-12-30"
 
 >> Text(Blank(),"mmm ddd yyy")
-"Dec Sat 1899"
\ No newline at end of file
+"Dec Sat 1899"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.LongDateTime)
+"Tuesday, August 15, 2023 2:53:09 AM"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.LongDate)
+"Tuesday, August 15, 2023"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.LongDateTime24)
+"Tuesday, August 15, 2023 2:53:09"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.LongTime)
+"2:53:09 AM"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.LongTime24)
+"2:53:09"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.ShortDate)
+"8/15/2023"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.ShortTime)
+"2:53 AM"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.ShortTime24)
+"2:53"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.ShortDateTime)
+"8/15/2023 2:53 AM"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), DateTimeFormat.ShortDateTime24)
+"8/15/2023 2:53"
+
+>> Text(44928, DateTimeFormat.LongDate)
+"Monday, January 2, 2023"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "DateTimeFormat.LongDateTime")
+"15ateTi8eFor8at.Long15ateTi8e"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'DateTimeFormat.LongDateTime'")
+"'15ateTi8eFor8at.Long15ateTi8e'"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "High'there")
+"2ig2't2ere"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "ZZZ 'longdatetime' ZZZ")
+"ZZZ 'long15ateti8e' ZZZ"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'longdatetime'")
+"'long15ateti8e'"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'longdatetime' ZZZ")
+"'long15ateti8e' ZZZ"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "LongDateTime")
+"Long15ateTi8e"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "longdatetime")
+"long15ateti8e"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'utc'")
+"'utc'"
```

Comment:
new v1 behavior, Should be in:
#SETUP: PowerFxV1CompatibilityRules

Reply:
move to test files.